Classes/Controller/GenericModelController.php

```diff
@@ -143,13 +164,21 @@ protected function initializeListAction()
         $propertyMappingConfiguration = $this->arguments['page']->getPropertyMappingConfiguration();
         assert($propertyMappingConfiguration instanceof PropertyMappingConfiguration);
         $propertyMappingConfiguration->allowAllProperties();
+
+        $propertyMappingConfiguration = $this->arguments['filter']->getPropertyMappingConfiguration();
```

Comment:
Das kann denk ich auch in eine Helper-Funktion, die nur den argument name bekommt

Reply:
Erledigt.